sktime/annotation/tests/test_all_annotators.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+import numpy as np
+from sktime.annotation.base._base import BaseAnnotator
+
+
+class TestAnnotator(BaseAnnotator):
+    def __init__(self):
+        pass
+
+    def fit(self, Z, X=None):
+        return self
+
+    def transform(self, Z, X=None):
+        Zt = Z.copy()
+        Zt.iloc[:] = False
+        Zt.iloc[1] = True
+        return Zt
+
+
+test_annotator = TestAnnotator()
+
+
+def test_output_is_series():
+
+    data = pd.Series(range(5))
```

Comment:
```python
ANNOTATORS = all_estimators("annotator", return_names=False)

@pytest.mark.parametrize("Estimator", ANNOTATORS):
def my_test_function(Estimator):
  estimator = _construct_instance(Estimator)
  
  args = _make_args(estimator, "fit")
  estimator.fit(*args)
  # assert somthing

  args = _make_args(estimator, "transform")
  estimator.transform(*args)
  # assert something
```
* need to update `all_estimators(...)`
* need to update `_make_args(...)` to handle annotators as estimators